Bio/SeqIO/SffIO.py

```diff
@@ -1360,7 +1360,7 @@ def write_record(self, record):
         # Basics
         name = record.id.encode()
         name_len = len(name)
-        seq = record.seq.upper().encode()
+        seq = record.seq.upper().encode("ASCII")  # TODO - how to get bytes now?
```

Comment:
@mdehoon is there a clean way to get the bytes string directly from a ``Seq`` object now? Other than ``._data`` I mean.

Reply:
Yes: `bytes(record.seq)`

Reply:
How does that work internally?

Reply:
`bytes` calls the `__bytes__` method of the argument.

Reply:
And when the ``Seq`` object here doesn't have a ``__bytes__`` method, falls back on ``__getitem__``?

Reply:
Sorry, Seq does have __bytes__ now. Must have had an old copy open.